JayKay24/micro_learning_web_app
# View Sent link inside page

#### What does this PR do?

This PR adds functionality to view the sent link on the Micro-Learning application.

#### How can this be manually tested?

* Log In to the application
* Create a category.
* Set the category to active in the settings tab.
* Wait a day. Click on Today's link. You should be able to view the link in the same application.

#### Screenshots

![screen shot 2018-07-26 at 13 00 18](https://user-images.githubusercontent.com/20414083/43258344-23f0d790-90db-11e8-81fd-d4c5131bf7da.png)

#### What are the relevant Pivotal Tracker stories?

[#158485088](https://www.pivotaltracker.com/story/show/158485088)
